pkgs/tools/video/gopro/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, ffmpeg
+, imagemagick
+, makeWrapper
+, mplayer
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gopro";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "KonradIT";
+    repo = "gopro-linux";
+    rev = version;
+    sha256 = "0sb9vpiadrq8g4ag828h8mvq01fg0306j0wjwkxdmwfqync1128l";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    chmod +x gopro
+    cp gopro $out/bin
+    wrapProgram $out/bin/gopro \
+      --suffix-each PATH ":" "${ffmpeg}/bin ${imagemagick}/bin ${mplayer}/bin"
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Command line interface for processing media filmed on GoPro HERO 3, 4, 5, 6, and 7 cameras.";
+    homepage = https://github.com/KonradIT/gopro-linux;
```

Comment:
Hint: We might remove unquoted URLs. See RFC 45 ;)

Reply:
Yea, I knew about that.... not sure why i didn't